bitnami/kong/README.md

```diff
@@ -410,3 +417,9 @@ $ helm upgrade my-release bitnami/kong \
 ```
 
 > Note: you need to substitute the placeholders _[POSTGRESQL_PASSWORD]_ with the values obtained from instructions in the installation notes.
+
+### To 2.0.0
+
+PostgreSQL and Cassandra dependency versions were bumped to new major versions, `9.x.x` and `6.x.x` respectively. Both of these include breaking changes and hence backwards compatibility is no longer guaranteed.
+
+Depending on your backend, you may refer to the [PostgreSQL Upgrade Notes](https://github.com/bitnami/charts/tree/master/bitnami/postgresql#900) or the [Cassandra Upgrade Notes](https://github.com/bitnami/charts/tree/master/bitnami/cassandra#to-600) to properly migrate your data to this new version.
```

Comment:
> Depending on your backend

What do you mean? Please elaborate it properly.

---

Does Kong provides its own way to bump/restore the data?

Reply:
Kong could be integrated with either PostgreSQL or Cassandra as per user preference. The property `database` in values lets you choose between this two. I will update the README to make this more clear 👍.

> Does Kong provides its own way to bump/restore the data?

To the best of my knowledge, it does not provide with its own feature to restore backup/restore the data. As our Kong setup relies on DBs to store configuration, restoring them would be sufficient.